airflow/dags/dags.py

```diff
@@ -114,6 +115,24 @@ def sql_enrich_duplicates(schema_tbl, key_columns, order_by_columns):
     """
 
 
+def email_failure(context):
+    slack_url = os.environ.get("CALITP_SLACK_URL")
+    if slack_url is None:
+        print("Skipping email to slack channel. No CALITP_SLACK_URL in environment")
```

Comment:
is this supposed to return here?

Reply:
I was wondering that too, not sure why I forgot to comment. It probably should return at this point.